.travis.yml

```diff
@@ -1,26 +1,49 @@
+sudo: false
+
 language: php
 
 php:
  - 7.0
- - 7.1
- - 7.2
- - nightly
+# TODO re-enable before merging
+# - 7.1
+# - 7.2
+# - nightly
 
 before_script:
   - travis_retry composer self-update
-  - travis_retry composer install --no-interaction --prefer-source --dev
+  - travis_retry composer install --no-interaction --prefer-dist --dev
+  # Install box (https://box-project.github.io/box2/)
+  - composer global require 'kherge/box=~2.4' --prefer-dist
```

Comment:
there is a new (and much faster among other things) version for which you don't have to worry about installing only the non-dev dependencies either: https://github.com/humbug/box :)